src-vue/src/plugins/store.ts

```diff
@@ -502,3 +511,19 @@ async function _get_northstar_version_number(state: any) {
             state.northstar_state = NorthstarState.INSTALL;
         })
 }
+
+async function display_plugin_warning() {
+    // just wait for the end user to click ok button
+    await ElMessageBox.alert("This mod contains a plugin. Plugins CAN BE REALLY DANGEROURS since they have to access to your system!", "Warning");
```

Comment:
This should be localised. See https://github.com/R2NorthstarTools/FlightCore/blob/main/docs/DEVELOPMENT.md#internationalization on how to do it.

You only need to do the English version. @Alystrasz and I can do French/German respectively ^^